### PR TITLE
prebuild mac-arm64, linux-arm and linux-arm64

### PR DESCRIPTION
**Pull request by Julusian**
I havent tested it fully, but tests I have done locally look to be working

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,8 +29,8 @@
   "scripts": {
     "install": "node-gyp-build",
     "vendor": "ts-node ./vendor/download-freetype.ts",
-    "prebuild": "prebuildify --napi -t 12.0.0 -t 14.0.0 -t electron@6.0.0 -t electron@9.3.5 -t electron@12.0.0 --strip --tag-libc",
-    "prebuild-ia32": "prebuildify --napi --arch=ia32 -t 12.0.0 -t 14.0.0 -t electron@6.0.0 -t electron@9.3.5 -t electron@12.0.0 --strip --tag-libc",
```
Comment (aminya) [inline package.json:33]: Why did you remove the electron prebuilds?

**Review by aminya — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Julusian) [reply package.json:33]: A combination of:
* simplifying this because it was building many versions to then replace them with the ones defined later. It ends up only keeping the 14.0.0 and electron@12.0.0 versions
* In every other native lib project, since switching to napi none of them have had electron specific builds so why does this lib need them?
* I tested it with just a node prebuild and it worked fine for me. But I should probably test on more than linux, windows is likely to be the annoying one

Comment (Julusian) [reply package.json:33]: I have just checked on windows, and if I:
* Make sure the prebuild script has no targets specified
* `pnpm prebuild`
* Delete the build folder (to ensure it uses the prebuild copy)
* Make sure the prebuild folder only contains a 'node' version

Then I can run a test script which does a require of the lib, both via node and `npx cross-env ELECTRON_RUN_AS_NODE=1 npx electron test.js`, and they both run happily. I verified that the versions produced by `console.log(process.versions)` do not match my locally installed version, and are from electron.

From this I would conclude that there is no need to have separate prebuilds for node and electron

Comment (Julusian) [reply package.json:33]: @aminya any more thoughts on this?

Comment (aminya) [reply package.json:33]: I need to test this with Atom's font-viewer package.

**Comment by Julusian** [comment]
fyi I am going my own way with a fork https://www.npmjs.com/package/@julusian/freetype2
It has been happily running in production for months in electron on mac-arm64, mac-x64, linux-x64, windows-x64 and in node on all of those as well as linux-arm64. Part of https://github.com/bitfocus/companion
